### DISCUSSION TITLE
Visual Feedback for Stepper Disabled

### DISCUSSION THREAD

**Discussion by tome9111991** [Jyers/Marlin#629] [Ideas]
I think is stock in marlin firmware.
If Stepper Disabled the axis on bottom in display is blinking.
maybe you like to implemented

Just an idea 😀

**Comment by Jyers** [comment]
Yeah, I've been meaning to do this for a while. I coded the status area that's being used on stock marlin and still haven't fixed up the one on my own fork... I'll put this on the todo list.
